apps/web-app/nuxt.config.ts

```diff
@@ -13,6 +13,14 @@ export default defineNuxtConfig({
     '@nuxt/ui',
     '@vue-email/nuxt'
   ],
+  // ssr: false,
+  routeRules: {
+    '/**': { ssr: false },
+    '/redirect': { ssr: true },
+    '/join/**': { ssr: true },
+    '/login/**': { ssr: true },
+    '/': { ssr: true }
+  },
```

Comment:
we do we still have those?

Reply:
initial login/signup pages will be SSR rendered for a faster experience
and just the main app will be client side
we could set the whole app to be client side
but I was planning to pre-load all components client side when we add in offline mode
this would resolve that

Reply:
@cstrnt just confirming you're happy with this